bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -1,488 +1,185 @@
-# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Comment:
Nitpick
```suggestion
# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
```

Reply:
@iomaganaris: 2023 -> 2024 changes would be part of upstreaming merge I believe. So I will keep it as it is.